scripts/build-php.sh

```diff
@@ -4,15 +4,8 @@ PHP_INI_DIR=/usr/local/etc/php
 
 mkdir -p $PHP_INI_DIR/conf.d
 
-PHP_EXT_PATH=/usr/local/lib/php/extensions/no-debug-non-zts-$PHP_EXT_SUFFIX
-PHP_CFLAGS="-fstack-protector-strong -fpic -fpie -O2"
-PHP_CPPFLAGS="$PHP_CFLAGS"
-PHP_LDFLAGS="-Wl,-O1 -Wl,--hash-style=both -pie"
-PHP_URL="https://secure.php.net/get/php-$PHP_VERSION.tar.xz/from/this/mirror"
-PHP_ASC_URL="https://secure.php.net/get/php-$PHP_VERSION.tar.xz.asc/from/this/mirror"
-
 apt-get update && apt-get install -y --no-install-recommends \
-        autoconf build-essential ca-certificates curl dirmngr dpkg-dev file g++ gcc gettext gnupg2 \
+        autoconf build-essential ca-certificates curl dirmngr dpkg-dev file g++ gcc gettext gnupg2 jq \
```

Comment:
Had to add `jq` to be able to parse the JSON response from PHP.net to determine the revision numbers and sha256 hashes.